decide/lic11.test.cpp

```diff
@@ -0,0 +1,27 @@
+#include <cassert>
+#include <cstdlib>
+#include <decide/decide.hpp>
+#include <decide/lics.hpp>
+
+// Should be fulfilled because points
+// (50,50) and (4,4) results in X[j]-X[i]=4-50<0.
+void test_true_instance() {
```

Comment:
Might not be the best test since it does not catch off by one errors in counting the `gpts` consecutive intervening points

Reply:
Resolved